src/main/java/gregtech/loaders/MaterialInfoLoader.java

```diff
@@ -13,281 +14,369 @@
 import net.minecraft.init.Items;
 import net.minecraft.item.ItemStack;
 
-import static gregtech.api.GTValues.M;
-import static gregtech.api.GTValues.W;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import static gregtech.api.GTValues.*;
 import static gregtech.api.unification.material.Materials.*;
 import static gregtech.common.metatileentities.MetaTileEntities.LONG_DIST_FLUID_ENDPOINT;
 import static gregtech.common.metatileentities.MetaTileEntities.LONG_DIST_ITEM_ENDPOINT;
 
+// TODO, material amount for planks ore prefix so we can use wood planks instead of using ingots?
 public class MaterialInfoLoader {
 
     public static void init() {
         OreDictUnifier.registerOre(MetaBlocks.WIRE_COIL.getItemVariant(CoilType.CUPRONICKEL),
-                new ItemMaterialInfo(new MaterialStack(Materials.Cupronickel, M * 8), // double wire
-                        new MaterialStack(Materials.Bronze, M * 2), // foil
-                        new MaterialStack(Materials.TinAlloy, M)) // ingot
+                new ItemMaterialInfo(new MaterialStack(Cupronickel, OrePrefix.wireGtDouble.getMaterialAmount(Cupronickel) * 8), // double wire
```

Comment:
i feel like there should be a helper method for `MaterialsStack` to make these lines less ugly